Babylon/commands/api/workspaces/service/api.py

```diff
@@ -0,0 +1,76 @@
+import sys
+from logging import getLogger
+from typing import Optional
+
+from Babylon.utils.request import oauth_request
+
+logger = getLogger("Babylon")
+
+
+class WorkspaceService:
+
+    def __init__(self, state: dict, azure_token: str, spec: Optional[dict] = None):
+        self.spec = spec
+        self.state = state
+        self.azure_token = azure_token
+        self.url = state["api"]["url"]
+        self.organization_id = state["api"]["organization_id"]
+
+        if not self.url:
+            logger.error("API url not found")
+            sys.exit(1)
+        if not self.organization_id:
+            logger.error("organization_id not found")
+            sys.exit(1)
+
+    def get_all(self):
+        response = oauth_request(
+            f"{self.url}/organizations/{self.organization_id}/workspaces",
+            self.azure_token,
+        )
+        return response
+
+    def get(self):
+        workspace_id = self.state["api"]["workspace_id"]
```

Comment:
is it really needed to put it here instead of in the init ?

Reply:
you are right, apparently, it doesn't break other commands, I'll change it ! :+1: